docs/contributing/translations.md

```diff
@@ -68,8 +68,42 @@ _("Page {page_title} with status {status}".format(page_title=page.title, status=
 _(f"Page {page.title} with status {page.status_string}")
 ```
 
+For example (in template):
+  -   `some_tag` is a placeholder for any tag.
+  -   `arg_of_some_tag` is a placeholder for an argument of a tag.
+  -   `value_of_some_tag` is a placeholder for a value passed to a tag.
+  -   `filter` is a placeholder for a filter.
+
+```html
```

Comment:
```suggestion
```html+django
```

This way the formatter and syntax highlighting works correctly.

Reply:
Ah, thank you! I didn't know about this.